test/test_render_notebook.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+import os
+import subprocess
+
+IMAGE = "docker-onecodex-notebook"
+
+
+def run_docker_container(tmp_path, container_command):
+    command = [
+        "docker",
+        "run",
+        "--rm",
+        *("--volume", f"{os.getcwd()}/test/notebooks/:/share"),
+        *("--user", "1000"),
+        *("--entrypoint", "/usr/local/bin/jupyter"),
+        IMAGE,
+        *container_command,
+    ]
+
+    result = subprocess.run(command, capture_output=True)
+
+    if result.returncode != 0:
+        raise Exception(
+            "\n".join(
+                [
+                    "Command Failed!",
+                    "command:",
+                    " ".join(command),
+                    "stdout:",
+                    result.stdout.decode("utf-8"),
+                    "stderr:",
+                    result.stderr.decode("utf-8"),
+                ]
+            )
+        )
+
+    return result
+
+
+def test_render_notebook(tmp_path):
+    run_docker_container(
+        tmp_path, ["nbconvert", "--execute", "--to", "onecodex_pdf", "/share/example.ipynb"]
```

Comment:
nit: Could you add a short comment about the existence of `onecodex_pdf` as a format?

Reply:
Done.